src/components/OrderConfirmation/PriceInfo.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from "prop-types";
+
+import { localizeMoney } from "../../lib/util";
+
+export const PriceInfo = ({ amount, currency, discount }) => {
+  const regularPrice = localizeMoney(amount / 100, currency);
+
+  if (!discount) {
+    return (
+      <div>
+        <strong>{regularPrice}</strong>
+      </div>
+    );
+  }
+  const { amountOff, duration, durationInMonths, percentOff } = discount;
+
+  const discountPrice = amountOff ? amount - amountOff : amount * (percentOff / 100);
```

Comment:
I would call `localizeMoney` on it here instead of in the return block. You seem to be calling `localizeMoney` without the `currency` for the discount price. Depending on the implementation of `localizeMoney` and the values with which you tested, you might have to ensure that the output of that method is always rounded to two decimals?

Reply:
`localizeMoney` actually defaults to `USD`. You're right about the rounding, though. I created a follow-up issue: https://linear.app/skynetlabs/issue/SKY-1392/round-the-purchase-total-on-order-confirmation-page